__tests__/plugins/network.test.ts

```diff
@@ -265,9 +265,8 @@ describe('network', () => {
       waitUntil: 'networkidle',
     });
     await driver.page.reload({ waitUntil: 'networkidle' });
-    await delay(delayTime);
+    await delay(delayTime + 1);
     const netinfo = await network.stop();
-    await Gatherer.stop();
```

Comment:
dont we need this?

Reply:
This causes a test failure with node 18, it's actually  double close, something closes it earlier

Reply:
Actually, I take it back, this test is acting funky with and without this. Could you take a look when you have a second?